app/controllers/admin/application_controller.rb

```diff
@@ -1,12 +1,12 @@
 class Admin::ApplicationController < ApplicationController
-  before_action :authorized?
+  before_action :authenticate_user!
+  before_action :authenticate_admin!
 
   private
 
-  def authorized?
-    return if current_user.admin?
+  def authenticate_admin!
+    return if current_user.try(:admin?)
```

Comment:
new syntax is `current_user&.admin?`
btw, here you don't need it

Reply:
it was substituted.